analyzer/protocol/ipsec/consts.zeek

```diff
@@ -0,0 +1,767 @@
+# Copyright (c) 2021 by the Zeek Project. See LICENSE for details.
+
+# All values from https://www.iana.org/assignments/ikev2-parameters/ikev2-parameters.xhtml
+# Portions have been copied from https://github.com/ukncsc/zeek-plugin-ikev2/blob/master/scripts/consts.zeek
```

Comment:
Reusing the code from the old analyzer is fine (it's BSD) but we need to be a bit careful with attribution, and also retain the original license as well. Can you mark the pieces that you reused, and then maybe we could put a copy of their license into the IPSec directory?

Reply:
It's the data in this file, plus I added to it as I found some missing values.  How do I mark that?

Reply:
Traces:  two of them I believe I pulled from the public site cloudshark.org.  I think two of the other ones came from the wireshark tests.

IPSec pcaps are something I wouldn't be able to easily generate on my own.

Reply:
I added additional_license.txt with a note in it.

Reply:
Links to traces:

https://www.cloudshark.org/captures/9e63e31f9f56?filter=frame%20and%20eth%20and%20ip%20and%20udp%20and%20udpencap

https://github.com/wireshark/wireshark/blob/master/test/captures/ikev1-certs.pcap

https://www.cloudshark.org/captures/ff740838f1c2

https://www.cloudshark.org/captures/e51f5c8a6b24

Reply:
It would be super neat if you could also refer the license in the comments.

I am not sure if we can add the traces if they don't have explicit licenses attached to them.

Reply:
Without the traces I won't have any tests for this.

Reply:
I added the trace source information in any directory someone would have to go in to use/compile ipsec here.  If someone wants the original source they can get it now, and also reference it like I did.